v3/tasks/20250313-v3-quantamm/index.ts

```diff
@@ -35,18 +37,19 @@ export default async (task: Task, { force, from }: TaskRunOptions = {}): Promise
   await task.save({ ChainlinkUsdcOracle: chainlinkUsdcOracleWrapper });
 
   console.log('Deployed ChainlinkUsdcOracle');
-  const accounts = (await hre.ethers.getSigners()) as unknown as string[];
+  const accounts = await hre.ethers.getSigners();
```

Comment:
Consider defining `adminAddress = accounts[0].address` for clarity and to future-proof against forgetting ".address"